requirements_dev.txt

```diff
@@ -1,5 +1,6 @@
 pytest-asyncio
 pytest-cov
+pytest-mock
```

Comment:
I'm not sure this is needed as python is new enough to use `unittest.mock` instead

Reply:
Not familiar with that library, will look it up and give it a go.